lib/bundler/postit_trampoline.rb

```diff
@@ -49,6 +49,15 @@ module BundlerVendoredPostIt; end
   nil
 end
 
+if Gem::Version.new(running_version) >= Gem::Version.new("1.13.0.rc.1".dup)
```

Comment:
Pretty sure this should be `if Gem::Requirement.new(">= 1.13.pre").satisfied_by?(Gem::Version.new(running_version))`.